studio/vite-plugin.ts

```diff
@@ -0,0 +1,105 @@
+import path from 'node:path';
```

Comment:
This vite plugin basically reads LOCAL_DOCUMENT_MODELS_IMPORT and LOCAL_DOCUMENT_EDITORS_IMPORT, resolves them into a filesystem path and sets in `resolve.alias`.

On the code we have a conditional `import("LOCAL_DOCUMENT_MODELS")` which resolves to that path. Vite then works it's magic and loads the package from that path, even if it's on typescript.